client/src/i18n.js

```diff
@@ -28,6 +30,7 @@ class i18n {
    */
   setLocale(locale) {
     this.currentLocale = locale;
+    this.autoDetectLocale = false;
```

Comment:
If someone explicitly sets a locale, we don't want to override their decision whenever a new dictionary is added.